packages/arb-token-bridge-ui/tests/support/commands.ts

```diff
@@ -8,40 +8,55 @@
 // ***********************************************
 
 import '@testing-library/cypress/add-commands'
-import { NetworkType, setupMetamaskNetwork, startWebApp } from './common'
+import {
+  l1NetworkConfig,
+  NetworkType,
+  setupMetamaskNetwork,
+  startWebApp
+} from './common'
 
-export function login(networkType: NetworkType) {
-  setupMetamaskNetwork(networkType).then(() => {
+export function login(networkType: NetworkType, addNewNetwork?: boolean) {
+  setupMetamaskNetwork(networkType, addNewNetwork).then(() => {
     startWebApp()
   })
 }
 
 // once all assertions are run, before test exit, make sure web-app is reset to original
 export const logout = () => {
   cy.switchToCypressWindow().then(() => {
-    // disconnect-metamask-wallet hangs if already not connected to metamask,
-    // so we do it while logout instead of before login.
-    cy.disconnectMetamaskWalletFromAllDapps().then(() => {
-      cy.switchToCypressWindow().then(() => {
-        cy.resetMetamaskAccount()
+    cy.changeMetamaskNetwork(l1NetworkConfig).then(() => {
+      // disconnect-metamask-wallet hangs if already not connected to metamask,
```

Comment:
didn't see this line of comment before
should we file an issue at the Synpress repo?

Reply:
I've added this as an issue there - https://github.com/Synthetixio/synpress/issues/529

Reply:
wrong link https://github.com/Synthetixio/synpress/issues/534 ^_^|||